src/pages/LandingPage/index.jsx

```diff
@@ -20,7 +23,9 @@ export const LandingPage = () => {
             <p className="text-wrapper-2">Your path to employment: streamlined job search guidance</p>
             <p className="text-wrapper-3">Feeling overwhelmed with job searching?</p>
           </div>
-          <Buttons buttonsSolidLargeText="Start Now" extraClasses="buttons-instance" property1="solid-large-blue" />
+          <div onClick={()=> navigateTo('/roadmap')}>
```

Comment:
Instead of using an extra `div`, update your `Buttons` component to take an extra (but optional) parameter and add the `onClick` event handler there. I've added a suggested change to the file